default.nix

```diff
@@ -128,6 +129,33 @@ rec {
     import ./templates/project { inherit lib pkgs sources; }
   );
 
+  pre-commit-hook = pre-commit-hooks.run {
+    src = ./.;
+    hooks = {
+      actionlint.enable = true;
+      nixfmt-rfc-style.enable = true;
+      actionlint.package = pkgs.actionlint;
+      nixfmt-rfc-style.package = pkgs.nixfmt-rfc-style;
```

Comment:
Note that without this, another version of `nixfmt-rfc-style` is used and formatting is not the same.